### PR TITLE
Update Tippy example in README.md

### PR DESCRIPTION
**Pull request by thehappydinoa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,17 @@ require(['cytoscape', 'cytoscape-popper'], function( cytoscape, popper ){
...
+<script src="https://unpkg.com/@popperjs/core@2/dist/umd/popper.min.js"></script>
+<script src="https://unpkg.com/cytoscape-popper@2.0.0/cytoscape-popper.js"></script>
+```
+
+For tippy:
+
+```html
+<script src="https://unpkg.com/tippy.js@6/dist/tippy-bundle.umd.min.js"></script>
+```
+
```
Comment (maxkfranz) [inline README.md:65]: Could suggesting specific CDN links make novice users expect that those are the only links that would work (only those specific versions etc.)?

Would the demo HTML files be a better place to comment on / document these links?

File: README.md [hunk 2]
```diff
@@ -154,7 +165,7 @@ let ref = node.popperRef(); // used only for positioning
 // https://atomiks.github.io/tippyjs/v6/constructor/#target-types
 let dummyDomEle = document.createElement('div');
 
-let tip = new Tippy(dummyDomEle, { // tippy props:
+let tip = new tippy(dummyDomEle, { // tippy props:
```
Comment (maxkfranz) [inline README.md:168]: OK

File: README.md (hunk 1 above)
Comment (thehappydinoa) [reply README.md:65]: Yeah that could definitely help, maybe we could just link to the demo file rather than the demo here?

Comment (maxkfranz) [reply README.md:65]: Good idea.  There are demo links at the top, but maybe they could be also linked in the relevant sections to make them more noticeable.

**Pull request closed without merging**